File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/RowBasedCommand_ViewModelMyComboBoxesRow.cs
/// <filename>
///     RowBasedCommand_ViewModelMyComboBoxesRow.cs
/// </filename>
using System.Collections.Generic;

namespace commands
{
    public class RowBasedCommand_ViewModelMyComboBoxesRow
  {
        private System.Collections.Generic.List<string> FreeTextComboBoxesComboBoxEntries = new System.Collections.Generic.List<string>();
     
        public     virtual System.Collections.Generic.List<string> getFreeTextComboBoxesComboBoxEntries()
    {
      return this.FreeTextComboBoxesComboBoxEntries;
    }
     
        private string FreeTextComboBoxesComboBoxSelectedEntry;
     
        public     virtual string getFreeTextComboBoxesComboBoxSelectedEntry()
    {
      return this.FreeTextComboBoxesComboBoxSelectedEntry;
    }
     
        private string FreeTextComboBoxesComboBoxText;
     
        public     virtual string getFreeTextComboBoxesComboBoxText()
    {
      return this.FreeTextComboBoxesComboBoxText;
    }
     
        private System.Collections.Generic.List<string> EntryComboBoxesComboBoxEntries = new System.Collections.Generic.List<string>();
     
        public     virtual System.Collections.Generic.List<string> getEntryComboBoxesComboBoxEntries()
    {
      return this.EntryComboBoxesComboBoxEntries;
    }
     
        private string EntryComboBoxesComboBoxSelectedEntry;
     
        public     virtual string getEntryComboBoxesComboBoxSelectedEntry()
    {
      return this.EntryComboBoxesComboBoxSelectedEntry;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/RowBasedCommand_ViewModelMyComboBoxesRow.cpp

#include "RowBasedCommand_ViewModelMyComboBoxesRow.h"
#include <vector>
#include <string>

namespace commands
{
  std::vector<std::string> RowBasedCommand_ViewModelMyComboBoxesRow::getFreeTextComboBoxesComboBoxEntries( ) 
  {
    return this->FreeTextComboBoxesComboBoxEntries;
  }
  std::string RowBasedCommand_ViewModelMyComboBoxesRow::getFreeTextComboBoxesComboBoxSelectedEntry( ) 
  {
    return this->FreeTextComboBoxesComboBoxSelectedEntry;
  }
  std::string RowBasedCommand_ViewModelMyComboBoxesRow::getFreeTextComboBoxesComboBoxText( ) 
  {
    return this->FreeTextComboBoxesComboBoxText;
  }
  std::vector<std::string> RowBasedCommand_ViewModelMyComboBoxesRow::getEntryComboBoxesComboBoxEntries( ) 
  {
    return this->EntryComboBoxesComboBoxEntries;
  }
  std::string RowBasedCommand_ViewModelMyComboBoxesRow::getEntryComboBoxesComboBoxSelectedEntry( ) 
  {
    return this->EntryComboBoxesComboBoxSelectedEntry;
  }
  int RowBasedCommand_ViewModelMyComboBoxesRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
}

